### DISCUSSION TITLE
Dropin Eclipse for Spotbugs

### DISCUSSION THREAD

**Discussion by VenomNerva** [spotbugs/spotbugs#2388] [Q&A] [answered]
Hello, did someone find out how to use the Dropin folder for Spotbugs ? Because there is not "features" or "plugins" folders similary to others plugins. 
And for my project I need to find a way to install some plugins without internet just by having the .zip or .tar

Thanks for the answer.

**Comment by hazendaz** [comment, marked as answer]
You don't need spotbugs for this, in fact you should use the following: https://github.com/m2e-code-quality/m2e-code-quality which covers spotbugs and many others and works really well for that purpose.
